com.obsproject.Studio.Plugin.VerticalCanvas.yml

```diff
@@ -0,0 +1,30 @@
+id: com.obsproject.Studio.Plugin.VerticalCanvas
+branch: stable
+runtime: com.obsproject.Studio
+runtime-version: stable
+sdk: org.kde.Sdk//6.4
+build-extension: true
+separate-locales: false
+appstream-compose: false
+build-options:
+  prefix: /app/plugins/VerticalCanvas
+modules:
+  - name: aitum
```

Comment:
Nitpick: `name: obs-vertical-canvas` (just to match the common practice, changing the module name doesn't make any effective difference)